dotenv/cut.go

```diff
@@ -0,0 +1,9 @@
+//go:build go1.18
```

Comment:
Does this work for everything greater than 1.18?

Reply:
Yep! Weird syntax but it's a minimum version opt-in. And thankfully it's gone now as of 3d146c71c3e41caafb84f23e3e84f77e4d83edd2!